app/views/proposals/new.html.haml

```diff
@@ -58,9 +58,9 @@
                 %span#abstract-maximum-word-count
                   250
                 words.
-
-              - if @program.cfp.enable_registrations?
-                = f.input :require_registration, label: 'Require participants to register to your event'
+              - if !@program.cfp.nil?
+                - if @program.cfp.enable_registrations?
```

Comment:
How do you even reach this page without a call for papers?

Reply:
@hennevogel "Submit proposal" button redirected to "proposals#new" but since there was no cfp, so i got the error. I have removed the button so a user will not be able to reach to this page. This code isn't required :sweat_smile: I will remove it. Sorry :sweat_smile: